filebeat/module/iis/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- key: iis
+  title: "iis"
+  description: >
+    iis Module
```

Comment:
For consistency with other Filebeat modules, I think this description should read:

> Module for parsing IIS log files.